cookbook/specs/stride.json

```diff
@@ -0,0 +1,1848 @@
+                    "parser_arg": [
+                      "latest"
+                    ],
+                    "parser_func": "DEFAULT"
+                  },
+                  "compute_units": 10,
+                  "enabled": true,
+                  "category": {
+                    "deterministic": true,
+                    "local": false,
+                    "subscription": false,
+                    "stateful": 0
+                  },
+                  "extra_compute_units": 0
+                },
+                {
+                  "name": "stride.staketia.Query/SlashRecords",
+                  "block_parsing": {
+                    "parser_arg": [
+                      "latest"
+                    ],
+                    "parser_func": "DEFAULT"
+                  },
+                  "compute_units": 10,
+                  "enabled": true,
+                  "category": {
+                    "deterministic": true,
+                    "local": false,
+                    "subscription": false,
+                    "stateful": 0
+                  },
+                  "extra_compute_units": 0
+                },
+                {
+                  "name": "stride.staketia.Query/UnbondingRecords",
+                  "block_parsing": {
+                    "parser_arg": [
+                      "latest"
+                    ],
+                    "parser_func": "DEFAULT"
+                  },
+                  "compute_units": 10,
+                  "enabled": true,
+                  "category": {
+                    "deterministic": true,
+                    "local": false,
+                    "subscription": false,
+                    "stateful": 0
+                  },
+                  "extra_compute_units": 0
+                },
+                {
+                  "name": "vesting.v1.Query/Balances",
+                  "block_parsing": {
+                    "parser_arg": [
+                      "latest"
+                    ],
+                    "parser_func": "DEFAULT"
+                  },
+                  "compute_units": 10,
+                  "enabled": true,
+                  "category": {
+                    "deterministic": true,
+                    "local": false,
+                    "subscription": false,
+                    "stateful": 0
+                  },
+                  "extra_compute_units": 0
+                }
+              ],
+              "headers": [],
+              "inheritance_apis": [],
+              "parse_directives": [],
+              "verifications": [
+                {
+                  "name": "chain-id",
+                  "values": [
+                    {
+                      "expected_value": "stride-1"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 14900
```

Comment:
we are missing archive verification here

Reply:
Resolved does the archive verification need to be applied on all i did it only on tendermint?

Reply:
yes, should be applied on all of the interfaces